### DISCUSSION TITLE
Support for Teams meetings

### DISCUSSION THREAD

**Discussion by AA87-pub** [code4recovery/12-step-meeting-list#1563] [Q&A]
One of the meetings in my Area is now meeting on Microsoft Teams and requested that I reflect the change on our meeting list. However, as I attempted to add the link to the meeting page, I noticed that it's not a supported provider.

Does anyone know if the developers are working on adding Teams to the supported online meeting providers? Or if a specific issue is preventing them from doing it? Alternatively, is there a work-around to add the meeting information anyway?

Thank you for your help!

**Comment by joshreisner** [comment]
thanks for bringing it to our attention! i'll create an issue and we should be able to get it added in the next few days!

**Reply by joshreisner** [reply]
hi @AA87-pub - we've rolled out support for Teams in the latest version (`3.16.4`) ✅
